app/routes.py

```diff
@@ -1,2 +1,209 @@
-from flask import Blueprint
+from flask.globals import session
+from app import db
+from app.models.customer import Customer
+from app.models.video import Video
+from app.models.rentals import Rental
+from flask import Blueprint, request, make_response, jsonify
+from flask_sqlalchemy import SQLAlchemy
+from datetime import datetime, timedelta
+from dotenv import load_dotenv
 
+load_dotenv()
+
+# ===== Customers ===================================================
+customers_bp = Blueprint("customer", __name__, url_prefix="/customers")
+
+@customers_bp.route("", methods=["GET"])
```

Comment:
I recommend adding strict_slashes=False to all of the routes, this will make the route work for both <url>/customers/ and <url>/customers.
```suggestion
@customers_bp.route("", methods=["GET"], strict_slashes=False)
```